api/app/signals/apps/api/generics/mixins.py

```diff
@@ -11,8 +11,8 @@ def convert_validation_error(error):
     Convert a Django ValidationError to a DRF ValidationError.
     """
     # TODO: handle Django ValidationError properties other than message
-    if hasattr(error, 'message'):
-        return DRFValidationError(error.message)
+    if hasattr(error, 'messages'):
```

Comment:
Make it an elif. And add a test for it.

Reply:
if hasattr(error, 'message'): 
    return DRFValidationError(error.messages)
elif hasattr(error, 'messages'): 
    return DRFValidationError(error.messages)
else:
    return DRFValidationError('Validation error on underlying data.')

Reply:
@Nick-amsterdam I added the changes myself. I will merge the PR when all checks are ✅